kclvm/config/src/settings.rs

```diff
@@ -231,3 +231,56 @@ mod settings_test {
         Ok(())
     }
 }
+
+pub fn build_settings_pathbuf(
```

Comment:
1. Using `build_settings_pathbuf` and `build_settings_pathbuf_with_matches`
2. Put `build_settings_pathbuf` into the `kclvm_config` crate